crates/virtio-queue/Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = "2018"
 test-utils = []
 
 [dependencies]
-vm-memory = "0.7.0"
+vm-memory = { version = "0.7.0", features = ["backend-mmap"] }
```

Comment:
Why is this dependency needed? We should not depend on this feature in the implementation, we're just using it in the tests so that custom VMMs can have their on vm-memory implementation

Reply:
I added this dependency because without it, when building with cargo build --release, I received an error:
error[E0432]: unresolved import `vm_memory::GuestMemoryMmap` from crates/virtio-queue/src/state.rs:14:60.